sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt

```diff
@@ -170,13 +163,9 @@ LogicalTableScan(table=[[sensors_2]])
     "CREATE VIEW `table$13`\nAS\nSELECT `sensorid`, `window_time` AS `timeSec`, AVG(`temperature`) AS `temp`\nFROM TABLE(TUMBLE(TABLE `table$12`, DESCRIPTOR(`timestamp`), INTERVAL '1' SECOND(4), INTERVAL '0' SECOND(1))) AS `t0`\nGROUP BY `sensorid`, `window_start`, `window_end`, `window_time`;",
     "CREATE VIEW `table$14`\nAS\nSELECT *\nFROM `sensorreading_1`;",
     "CREATE VIEW `table$15`\nAS\nSELECT *\nFROM `sensors_1`;",
-    "CREATE VIEW `table$16`\nAS\nSELECT *\nFROM (SELECT `id`, `machineId`, `placed`, `timestamp`, ROW_NUMBER() OVER (PARTITION BY `id` ORDER BY `timestamp` DESC) AS `_rownum`\n  FROM `sensors_1`) AS `t`\nWHERE `_rownum` = 1;",
-    "CREATE VIEW `table$17`\nAS\nSELECT `sensorid`, ENDOFSECOND(`timestamp`) AS `timeSec`, `temperature`, `time`, `timestamp`\nFROM `sensorreading_1`;",
-    "CREATE VIEW `table$18`\nAS\nSELECT `t0`.`machineId` AS `machineid`, `$cor9`.`sensorid`, `$cor9`.`temp`, `$cor9`.`timeSec`\nFROM (SELECT `sensorid`, `timeSec`, `temp`, `sensorid` AS `sensorid3`\n  FROM (SELECT `sensorid`, `window_time` AS `timeSec`, AVG(`temperature`) AS `temp`\n    FROM TABLE(TUMBLE(TABLE `table$17`, DESCRIPTOR(`timestamp`), INTERVAL '1' SECOND(4), INTERVAL '0' SECOND(1))) AS `t2`\n    GROUP BY `sensorid`, `window_start`, `window_end`, `window_time`) AS `t4`\n  WHERE `temp` > 35) AS `$cor9`\n INNER JOIN `table$16` FOR SYSTEM_TIME AS OF `$cor9`.`timeSec` AS `t0` ON `t0`.`id` = `$cor9`.`sensorid3`;",
-    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `hightemp_1`\n(SELECT *\n FROM `table$3`)\n;\nINSERT INTO `lasthour_1`\n (SELECT *\n  FROM `table$7`)\n ;\n INSERT INTO `machine_1`\n  (SELECT *\n   FROM `table$11`)\n  ;\n  INSERT INTO `secreading_1`\n   (SELECT *\n    FROM `table$13`)\n   ;\n   INSERT INTO `sensorreading_2`\n    (SELECT *\n     FROM `table$14`)\n    ;\n    INSERT INTO `sensors_3`\n     (SELECT *\n      FROM `table$15`)\n     ;\n     INSERT INTO `hightemp_1_1`\n      (SELECT *\n       FROM `table$18`)\n      ;\n      END;"
+    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `hightemp_1`\n(SELECT *\n FROM `table$3`)\n;\nINSERT INTO `lasthour_1`\n (SELECT *\n  FROM `table$7`)\n ;\n INSERT INTO `machine_1`\n  (SELECT *\n   FROM `table$11`)\n  ;\n  INSERT INTO `secreading_1`\n   (SELECT *\n    FROM `table$13`)\n   ;\n   INSERT INTO `sensorreading_2`\n    (SELECT *\n     FROM `table$14`)\n    ;\n    INSERT INTO `sensors_3`\n     (SELECT *\n      FROM `table$15`)\n     ;\n     END;"
   ],
   "connectors" : [
-    "print",
```

Comment:
Why did you have to remove the export to print from this test?

Reply:
the thousandish print statements were making the job slow

Reply:
👍